frontend/public/locales/en/details-page.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Details": "Details",
+  "Logs": "Logs"
+}
```

Comment:
It seems it's not used anywhere now. Run yarn i18n and that should remove these strings. Also remove this file.